File: Kotlin/pangram.kt
fun main() {
    val sentence = input()
    output(sentence)
}

fun input(): String = readLine() ?: ""

fun output(sentence: String) {
    print(check(sentence))
}

fun check(sentence: String): Boolean {
    val letters = mutableMapOf<Char, Int>()
    
    for (s in sentence) {
        letters[s] = letters.getOrDefault(s, 0) + 1
    }
    
    var isPan = true
    for (c in 'a'..'z') {
        if (letters.getOrDefault(c, 0) == 0) {
            println("There is no $c")
            isPan = false
        }
    }
    return isPan
}

File: Swift/pangram.swift
func input(_ sentence: inout String) {
    if let line = readLine() {
        sentence = line
    }
}

func output(_ sentence: String) {
    print(check(sentence))
}

func check(_ sentence: String) -> Bool {
    var letters: [Character: Int] = [:]
    for s in sentence {
        letters[s, default: 0] += 1
    }
    var isPan = true
    for ascii in 97...122 {
        let c = Character(UnicodeScalar(ascii)!)
        if letters[c, default: 0] == 0 {
            print("There is no \(c)")
            isPan = false
        }
    }
    return isPan
}

var sentence = ""
input(&sentence)
output(sentence)